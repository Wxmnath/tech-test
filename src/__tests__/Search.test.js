import { render } from "@testing-library/react";
import SearchForm from "../components/Search";

test("renders learn react link", () => {
  render(<SearchForm />);
  const { asFragment } = render(<SearchForm />);
  expect(asFragment).toMatchSnapshot();
});
