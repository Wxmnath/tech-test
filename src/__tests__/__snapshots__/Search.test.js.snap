// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search renders learn react link 1`] = `[Function]`;
